Source/monster.cpp

```diff
@@ -5074,10 +5086,9 @@ bool SpawnSkeleton(int ii, Point position)
 		}
 	}
 
-	int dx = position.x - 1 + x2;
-	int dy = position.y - 1 + y2;
-	Direction dir = GetDirection({ dx, dy }, position);
-	ActivateSpawn(ii, dx, dy, dir);
+	Point spawn = position + Displacement { x2 - 1, y2 - 1 };

```

Comment:
😑 I see what this is doing now, everything from 5072 to here is trying to find a free adjacent spot in a given Direction. This cleanup made it easy to spot at least 😅